lettre ouverte n2 assos

simenca opened the pull request:
![fb_unis_pour_bois_le_roi_torchon](https://user-images.githubusercontent.com/66877911/175395330-84e01285-9ac6-4149-b718-bdd908f2d2fd.png)

@toopy il faudrait ajouter cette image dans le corps de la lettre. Je ne sais pas faire. Peux tu le faire?
Merci.

simenca committed:
lettre ouverte n2 assos

simenca committed:
ajout copie ecran facebook

simenca commented:
@toopy Bon j'ai du merdé sur un truc car ca passe pas suremment une erreur sur le chemin du folder images mais je vois pas le pb

toopy commented on content/blog/lettre-ouverte-assos-collectifs-du-29-03-2022.md:
```diff
@@ -0,0 +1,42 @@
+draft: false
+weight: 50
+images: []
+contributors: ["Bois-le-Roi Environnement","Vive la démocratie à Bois-le-Roi",,"Collectif de l'arbre","Collectif des voisins de la Foccel","Bien à Bois-le-Roi","Comité de sauvegarde du quartier de la gare"]
```
```suggestion
contributors: ["Bois-le-Roi Environnement","Vive la démocratie à Bois-le-Roi","Collectif de l'arbre","Collectif des voisins de la Foccel","Bien à Bois-le-Roi","Comité de sauvegarde du quartier de la gare"]

```

toopy committed:
Fix syntac error in headers

toopy committed:
Use common images/ folder for blog posts

Or special characters in the page title will lead to not reachable
image links.

simenca commented:
@toopy C'est bon pour moi si tu peux valider la PR.
